frontend/lib/identification/verification_workflow/verification_qr_code_processor.dart

```diff
@@ -48,24 +49,21 @@ Future<CardInfo?> verifyStaticVerificationCode(
   String projectId,
   StaticVerificationCode code,
 ) async {
-  assertConsistentCardInfo(code.info);
+  CardInfo cardInfo = code.info;
+  assertConsistentCardInfo(cardInfo);
   _assertConsistentStaticVerificationCode(code);
+  _assertCardIsAlreadyValid(cardInfo);
+
   if (!(await queryStaticServerVerification(client, projectId, code)).valid) {
     return null;
   }
   return code.info;
 }
 
-void assertConsistentCardInfo(CardInfo cardInfo) {
```

Comment:
🙃 I think the location was fine here.
other assertions are here & the function is only used here